### PR TITLE
PMM-6681 Allow admin download logs

### PR DESCRIPTION
**Pull request by Dasio**
[PMM-6681](https://jira.percona.com/browse/PMM-6681)

- [x] Build: https://github.com/Percona-Lab/pmm-submodules/pull/1424

- [ ] Links to other linked pull requests (optional).
---
- [x] Tests passed.
- [ ] Feature build pass.
- [ ] (Re)requested review.
- [ ] Fix conflicts with target branch.
- [ ] Update API dependency.
- [ ] Update jira ticket description if needed.
- [ ] Attach screenshots/console output to confirm new behavior to jira ticket, if applicable.

### CODE REVIEW

File: services/grafana/auth_server.go [hunk 1]
```diff
@@ -70,7 +70,7 @@ var rules = map[string]role{
 	"/graph":           none,
 	"/qan":             none,
 	"/swagger":         none,
-
+	"/logs.zip":        admin,
 	// "/auth_request" and "/setup" have auth_request disabled in nginx config
```
Comment (AlekSi) [inline services/grafana/auth_server.go:74]: That group was intended to be the same as mustSetupRules.

Please add a comment on top of that, and keep a blank line before logs.zip.

Please also add "/victoriametrics" to mustSetupRules

Comment (BupycHuk) [reply services/grafana/auth_server.go:74]: +1

Comment (BupycHuk) [reply services/grafana/auth_server.go:76]: @AlekSi I couldn't find `/victoriametrics` in nginx

Comment (AlekSi) [reply services/grafana/auth_server.go:76]: https://github.com/percona/pmm-server/blob/f09e39bb7de10ca69174a9ad96445953aacf512f/nginx.conf#L122-L126

File: services/grafana/auth_server.go [hunk 2]
```diff
@@ -64,13 +64,15 @@ var rules = map[string]role{
...
 
+	// mustSetupRules group
 	"/prometheus":      admin,
 	"/victoriametrics": admin,
 	"/alertmanager":    admin,
 	"/graph":           none,
 	"/qan":             none,
 	"/swagger":         none,
 
+	"/logs.zip": admin,
```
Comment (BupycHuk) [inline services/grafana/auth_server.go:75]: based on the task description `admin:admin` is already has access to download logs.zip, so we should create a user with Admin level (but not Grafana Admin). I've updated description to explain how to create this type of user.

**Review by BupycHuk — CHANGES_REQUESTED** [review]

**Comment by AlekSi** [comment]
@Dasio please re-request review from @BupycHuk by clicking a button next to his name on the top-right

**Comment by BupycHuk** [comment]
Please include link to FB

**Review by BupycHuk — APPROVED** [review]

**Review by askomorokhov — APPROVED** [review]

**Pull request merged**